sycl/doc/KernelProgramCache.md

```diff
@@ -175,8 +175,9 @@ three sources of build options:
 There is set of configuration parameters which can be set as environment variables or parameters in `sycl.conf`  and affect cache behavior:
 | Environment variable | Values | Description |
 | -------------------- | ------ | ----------- |
```

Comment:
Can we avoid information duplication? It's going to be out of sync at some point.
Can probably have a section about SYCL_CACHE_* vars at EnvironmentVariables.md and some reference here.